service/src/main/java/bio/terra/policy/service/policy/PolicyLabel.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.policy.service.policy;
+
+import bio.terra.policy.common.model.PolicyInput;
+import bio.terra.policy.common.model.PolicyName;
+
+/** A policy that does not contain any additional data. */
+public class PolicyLabel implements PolicyBase {
+  private final PolicyName policyName;
+
+  public PolicyLabel(PolicyName policyName) {
+    this.policyName = policyName;
+  }
+
+  @Override
+  public PolicyName getPolicyName() {
+    return policyName;
+  }
+
+  /**
+   * Return either dependent or source, whichever is non-null, or null if both are null.
+   *
+   * @param dependent policy input
+   * @param source policy input
+   * @return policy input or null, if there is a conflict
+   */
+  @Override
+  public PolicyInput combine(PolicyInput dependent, PolicyInput source) {
+    return dependent == null ? source : dependent;
```

Comment:
Would this work to combine 2 different PolicyLabels? We're adding protected-data right now, if there's another type then we should be able to combine them. Maybe this is fine for now and we can cross that bridge when we add another label type.

Reply:
It would be a good defense to verify that the policy name matches.